CatalogReaderExample/NuGet.Protocol.Catalog.Sample/LoggerCatalogLeafProcessor.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class LoggerCatalogLeafProcessor : ICatalogLeafProcessor
+    {
+        private const int FailAfterCommitCount = 10;
+        private readonly ILogger<LoggerCatalogLeafProcessor> _logger;
+        private DateTimeOffset? _lastCommitTimestamp;
+        private int _commitCount;
+
+        public LoggerCatalogLeafProcessor(ILogger<LoggerCatalogLeafProcessor> logger)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            _lastCommitTimestamp = null;
+            _commitCount = 0;
+        }
+
+        public Task<bool> ProcessPackageDeleteAsync(PackageDeleteCatalogLeaf leaf)
+        {
+            Console.WriteLine(
```

Comment:
Should these `Console.WriteLine` be `_logger.Info`?

Reply:
I intentionally made the sample output different for these things so that it always should output (even when log settings disable console output, or whatever).